### DISCUSSION TITLE
Smart profiles

### DISCUSSION THREAD

**Discussion by albert-kozera** [streamdeck-linux-gui/streamdeck-linux-gui#138] [Ideas / Feature Requests]
Hi, There was already an issue about that in original project:
https://github.com/timothycrosley/streamdeck-ui/issues/215

Do you plan to implement that feature? That would be huge improvement!

**Comment by coolapso** [comment]
Well not directly by any maintainer, but also in this case this project is not directly driven by maintainers but rather by the community at large, if anyone is willing to work on it and implement it it will be for sure reviewed and accepted if it meets all requirements.

**Comment by ajbucci** [comment]
This is possible on X but not easily on Wayland. Wayland support would require a different implementation for each desktop environment / compositor. For example, on gnome/wayland the only way to get the active window title is using a gnome extension that makes it available via dbus, with no guarantees that it won't break with future gnome releases.
